File: 9041 - Software Instruction/test11/devowel_file.py
#!/usr/bin/python

import re,sys

filename = sys.argv[1]

input_flush = open(filename,'r')
array = list()
for line in input_flush.readlines():
	line = re.sub('[aeiouAEIOU]', '', line)
	array.append(line)

f = open(filename, 'w')
for string in array:
	f.write("%s" % string)
f.close()

File: 9041 - Software Instruction/test11/devowel_file.pl
#!/usr/bin/perl -w

$file = $ARGV[0];
open(my $fh, '<', $file) or die "Could not open file '$file' $!";
my @array;
foreach $line (<>) {
 $line =~ s/[aeiouAEIOU]//g;
 push(@array, $line);
}

open(my $f, '>', $file);
print $f @array;
close($f)
